### DISCUSSION TITLE
Reproject from one coordinate to another

### DISCUSSION THREAD

**Discussion by sigster** [DotSpatial/DotSpatial#1558] [Q&A]
Hi

How can I Reproject shapefile from one coordinate to another

1. open
2. Reproject
3. Saveas

Regards
Sigster

**Comment by sindizzy** [comment]
Use the Project() functions.

**Comment by sigster** [comment]
Thanks 

Any chance you know about example i can use

Thanks

**Comment by sindizzy** [comment]
In the repo there is a folder with examples and tests. There should be one for projecting to another coordinate system.
